jest.setup.js

```diff
@@ -13,6 +14,7 @@ global.TextEncoder = TextEncoder;
 global.TextDecoder = TextDecoder;
 
 Vue.config.productionTip = false;
+Vue.use(i18n);
```

Comment:
In Jest configuration with Vue3 I'd to do something like:
```

config.global.directives['t'] = t;
config.global.directives['clean-tooltip'] = VCleanTooltip ;
config.global.components['v-select'] = vSelect;
config.global.plugins = [VTooltip];
config.global.mocks['t'] = key => `%${key}%`
```

How would you translate that using i18n?
Using just `config.global.plugins = i18n`? 
Would that include all the 3 parts (global, component, directive)?

Reply:
That sounds good because I've to mock many parts instead of 1 only.